### ISSUE TITLE
Unused variable rule not working in abstract class

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#473]
This shows the error:
```
CLASS Test:
   METHOD PUBLIC STATIC VOID Foo(   ):
      DEFINE VARIABLE lcTest AS CHARACTER NO-UNDO.
   END METHOD.
END CLASS.
```

This doesn't show the error:
```
CLASS Test ABSTRACT:
   METHOD PUBLIC STATIC VOID Foo(   ):
      DEFINE VARIABLE lcTest AS CHARACTER NO-UNDO.
   END METHOD.
END CLASS.
```

**Comment by gquerret** [comment]
Fixed, merged in develop
